bundle/manifests/manager-config_v1_configmap.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+data:
+  controller_manager_config.yaml: |
+    # TODO: Determine what to do with this file
```

Comment:
How does this file get used? Please ping the SDK team for direction.

Reply:
Changed  this and added an explanation in config/manager/kustomization.yaml